# ==============================================================================
# FIXING COMMIT DIFF
# ==============================================================================
# File: layout/forms/crashtests/1600207.html
# Commit: 8056d04a14bc
# Full Hash: 8056d04a14bcff203cb3ceb1e692e083a4cd90b7
# Author: Mats Palmgren <mats@mozilla.com>
# Date: 2019-12-09 16:35:51
# Description:
#   Bug 1600207 - Make GetInner()/GetLegend() robust also in presence of additional continuations on the principal child list.  r=TYLin
#   
#   Also, don't drain OverflowList unless we need to.
#   And make EnsureChildContinuation deal with continuations going from being
#   normal continuations to overflow-continuations (and vice versa) better.
# ==============================================================================

diff -r f2078bae1c46 -r 8056d04a14bc layout/forms/crashtests/1600207.html
--- /dev/null	Thu Jan 01 00:00:00 1970 +0000
+++ b/layout/forms/crashtests/1600207.html	Mon Dec 09 11:24:50 2019 +0000
@@ -0,0 +1,9 @@
+<style>
+:not(isindex) {
+  columns: 0px;
+  margin-top: 1%;
+}
+</style>
+<fieldset>
+<output>
+<br></br>